operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1235,36 +1236,46 @@ spec:
                       value: "120000"
                     - name: STRIMZI_OPERATION_TIMEOUT_MS
                       value: "300000"
-                    - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+                    - name: RELATED_IMAGE_STRIMZI_CLUSTER_OPERATOR_IMAGE
+                      value: registry.redhat.io/amq-streams/strimzi-rhel9-operator@sha256:06de6eaeb8cae4ea31f05a427bbe53f9d078de0030bd56fce4ecfbb5d0be461a
+                    - name: RELATED_IMAGE_STRIMZI_KAFKA_IMAGES_3_6_0
+                      value: registry.redhat.io/amq-streams/kafka-36-rhel9@sha256:177484ebf6f663eedfc558285e4e89e817277389250aec29323452082b6949e4
+                    - name: RELATED_IMAGE_STRIMZI_KAFKA_IMAGES_3_7_0
                       value: registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:050aed7a91af406fbafcf6ce8fff4fa451cf4855f74a91aa081d794bf8c9aaa3
+                    - name: RELATED_IMAGE_STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+                      value: registry.redhat.io/amq-streams/bridge-rhel9@sha256:4db7b231d68a244f5f71c8587123890e1344be933e1a030097c4b32a9bbfe32d
+                    - name: RELATED_IMAGE_STRIMZI_DEFAULT_MAVEN_BUILDER
+                      value: registry.redhat.io/amq-streams/maven-builder-rhel9@sha256:2393070a07677a482e90f53fef58f12e1c73963cf164738aaa1c095e5fe77d0d
```

Comment:
I suppose these values will be plugged-in with image references when `render_templates` runs, correct?

IMO using annotations is a little cleaner because it doesn't touch the operator's runtime environment with additional variables. I'm fine if it works here though.

Reply:
> I suppose these values will be plugged-in with image references when render_templates runs, correct?

yes

> IMO using annotations is a little cleaner because it doesn't touch the operator's runtime environment with additional variables. I'm fine if it works here though.

Not sure I understand, are you suggesting we move the image references into the annotations and then reference them in the ENV VARs instead of using creating `RELATED_IMAGES` env vars?

Reply:
Right, that's an alternative to using `RELATED_IMAGES`. In that case you get a slightly different `relatedImages` name for each. I think it's something like `<annotation-name>-<image-sha>-annotatation`. My opinion on it is that is keeps the container's `env` list cleaner and limited to only the things that should really be in the environment.

Reply:
I'll give that a go

Reply:
Made changes in latest commit, let me know what you think!